wagtail_simple_gallery/models.py

```diff
@@ -122,8 +122,8 @@ def tag_archive(self, request, tag=None):
         return render(request, getattr(settings, 'SIMPLE_GALLERY_TEMPLATE', 'wagtail_simple_gallery/simple_gallery_index.html'), context)
 
     class Meta:
-        verbose_name = _('Gallery index')
-        verbose_name_plural = _('Gallery indices')
+        verbose_name = _(getattr(settings, 'SIMPLE_GALLERY_PAGE_TYPE', 'Gallery index'))
+        verbose_name_plural = _(getattr(settings, 'SIMPLE_GALLERY_PAGE_TYPE_PLURAL', 'Gallery indices'))
```

Comment:
I just took a look and I cannot find any place where the plural string would be visible. Do you fell like removing it completely?

Reply:
At least when clicking around in wagtail admin I couldn't find anything neither. Yeah, why not just remove it. Sometimes less is more. If someone's missing it, he'll find this change or open an issue on this repo. Fun fact sidenote: I also was thinking that the english plural of index is indices, but a little research reveals that also indexes is a valid plural form. But anyway and since I've done my homework and jead the Django docs, as you just mentioned "index+s" would be wrong. So to answer your question: Yes, I would be brave enough to remove the django setting and the verbose_name_plural entirely :-)

Reply:
Yeah makes sense. Wanna update the pr?